onnxruntime/test/optimizer/qdq_transformer_test.cc

```diff
@@ -21,102 +21,22 @@
 #include "gtest/gtest.h"
 #include "graph_transform_test_builder.h"
 
+#include "qdq_test_utils.cc"
```

Comment:
this might be problematic.. but just include .cc file here for now..

also not so sure under what directory would be good for these shared functions

Reply:
I think they're fine in where they are. 

the templatized functions would usually be in a .h file (which also makes the #include look far more normal) so I would rename qdq_test_utils.cc to qdq_test_utils.h.